smrf/tests/smrf_test_case.py

```diff
@@ -60,6 +117,9 @@ def assertGoldEqual(self, gold, not_gold, error_msg):
                 err_msg=error_msg
             )
 
+    def setUp(self):
+        self._dist_variables = None
```

Comment:
What is the thinking for setting this to `None` in order to copy the `DIST_VARIABLES`?

Reply:
By ensuring that all instances have a internal variable declared, you can do this kind of 'lazy loading' with a property:
```
    @property
     def dist_variables(self):
         if self._dist_variables is None:
```
Otherwise you could go the route of `getattr()`.

Preference?

Reply:
Works for me